test/e2e/flask/simulation-preview.mocks.ts

```diff
@@ -0,0 +1,90 @@
+import type { Mockttp } from "mockttp";
+
+const MOCK_SENDER_ADDRESS = '0x5cfe73b6021e818b776b421b1c4db2474086a7e1';
+export const MOCK_RECIPIENT_ADDRESS = '0xe18035bf8712672935fdb4e5e431b1a0183d2dfc';
+
+const nativeSend = {
```

Comment:
Since this is a constant, should we call it something like `SIMULATION_REQUEST_NATIVE_CHANGE_ONLY_MOCK`?

I appreciate it also includes the response, but request feels like an umbrella term.